packages/PlanLimitsUI/package.json

```diff
@@ -6,62 +6,58 @@
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview",
-    "prettier": "pretty-quick",
```

Comment:
What do you do for just "prettier all the things" ... or just don't have the need for it?